scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -146,6 +148,66 @@ public void updateTimeOfDay() {
     );
   }
 
+  /**
+   * Converts the LightingMode values to strings to display in the open ComboBox.
+   */
+  public class LightingModeListCell extends ListCell<LightingMode> {
+
+    @Override
+    protected void updateItem(LightingMode lightingMode, boolean empty) {
+
+      super.updateItem(lightingMode, empty);
+      if (lightingMode != null) {
+        if (lightingMode == LightingMode.LIGHT) setText("Light");
+        else if (lightingMode == LightingMode.LIGHT_AND_SHADOWS) setText("Light and shadows");
+        else if (lightingMode == LightingMode.NO_LIGHT) setText("No light");
+      }
+    }
+  }
+
+  /**
+   * Converts the LightingMode values to strings to display in the ComboBox.
+   */
+  public class ComboBoxStringConverter extends StringConverter<LightingMode> {
```

Comment:
these should be marked as `private` classes so that they are only accessible within the Controller.